2200057/1949.py

```diff
@@ -0,0 +1,45 @@
+T = int(input())
+dx = [1, 0, -1, 0]
+dy = [0, 1, 0, -1]
+
+for t in range(1, T+1):
+    N, K = map(int, input().split())
+    ground = []
+    max_h = 0
+    max_route = 0
+    visited = [[0] * N for _ in range(N)]
+
+    for i in range(N):
+        li = list(map(int, input().split()))
+        max_h = max(max(li), max_h)
+        ground.append(li)    
+
+    def dfs(x, y, construct):
+        global max_route, visited
```

Comment:
```suggestion
        global max_route
```
list 자료구조의 값을 수정하는 경우엔 글로벌 변수 선언을 하지 않아도 괜찮습니다 😄